react/orders101/src/pages/Home.jsx

```diff
@@ -0,0 +1,24 @@
+import Banner from "../features/banner/Banner";
+import Categories from "../features/categories/Categories";
```

Comment:
I was trying to run this locally but got a complaint about this path - seems like the folder is named with uppercase `C`. I assume it's just a mistake but perhaps it can be renamed with lowercase to be consistent with the others?

Reply:
Locally I have the folder name as 'categories' with a lowercase c. I'm not sure why that wasn't pushed through